Jahia/npm-modules-engine
# Backlog 22550 update folder structure

## Description

1) Add mappings for the new npm module folder structure .
With the new folder structure, most static files have changed of place, so I added a mapping that convert the new folder structure into the old one in the OSGi bundle.

2) Change the folder structure of the test-module according to the new specs

## Tests

1) Build the test-module
2) Import it in Jahia (with the lattest version of npm-module-engin + serverSettings) and check that there is no errors
3) Check that everything works in jahia : 
- content-type icons are visible when adding content to a page
- template preview image is visible when creating a site
- locales are working
- Logo and favicon are visible
- Client JS is working when logging
- content-editor-forms is working (adding an image to an agency is possible)
